specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json

```diff
@@ -1643,6 +1691,10 @@
           "$ref": "#/definitions/PrivateEndpointConnectionProperties",
           "x-ms-client-flatten": true,
           "description": "Resource properties."
+        },
+        "etag": {
```

Comment:
This is normally a header. Why part of the payload? Should people still pass it back in a request header? Needs more docs.

Reply:
The `etag` here varies per private endpoint connection, which is an array element of the vault properties. The same vault can have multiple connections, each with a different etag.

Currently the `etag` is informative only. It only helps users and tools to detect that there was some change in the connection. We ignore the value specified in PUT or POST requests.